src/frontend/Sider/Behandling/Stønadsvilkår/PassBarn/PassBarn.tsx

```diff
@@ -34,16 +38,28 @@ const PassBarn: React.FC<Props> = ({ vilkårsregler, vilkårsvurdering }) => {
             return <div>Fant ikke grunnlag for barn</div>;
         }
 
+        const barnetsNavn = grunnlagBarn.registergrunnlag.navn;
+        const barnetsAlder = grunnlagBarn.registergrunnlag.alder || '-';
+
         return (
-            <VilkårPanelMedResultat
-                tittel={grunnlagBarn.registergrunnlag.navn}
-                resultat={vilkår.resultat}
-                key={grunnlagBarn.barnId}
-                paragrafLenker={lovverkslenkerPassBarn}
-                rundskrivLenke={rundskrivPassBarn}
+            <EkspanderbartPanel
+                tittel={`${barnetsNavn} (${barnetsAlder} år)`}
+                ikon={<VilkårsresultatIkon vilkårsresultat={vilkår.resultat} />}
+                ekstraHeading={
+                    <>
+                        <HStack gap="1" align="center">
+                            <Label>{grunnlagBarn.ident}</Label>
+                            <CopyButton copyText={grunnlagBarn.ident} size="small" />
+                        </HStack>
+                        <ParagrafOgRundskrivLenker
+                            paragrafLenker={lovverkslenkerAktivitet}
```

Comment:
Nå brukes lovverkslenkene for pass av barn

Reply:
Nice catch! Fiksa